material/frontend/views/list.py

```diff
@@ -238,6 +239,10 @@ def format_column(self, item, field_name, value):
             return self.empty_value_display
         elif isinstance(value, datetime.datetime):
             return formats.localize(timezone.template_localtime(value))
+        elif isinstance(value, bool):
```

Comment:
I try to keep an HTML formatting out of DataTableMixin, and put it into ListModelView